client/extensions/belgaCoverage/src/belga-coverage-preview.tsx

```diff
@@ -2,28 +2,28 @@ import * as React from 'react';
 import {IPreviewComponentProps, ISuperdesk} from 'superdesk-api';
 
 import {parseIds} from './belga-coverage-editor';
-import BelgaCoverageInfo from './belga-coverage-info';
 import BelgaCoverageImages from './belga-coverage-images';
+import BelgaCoverageCarousel from './belga-coverage-carousel';
 
 export function getBelgaCoveragePreview(superdesk: ISuperdesk): React.StatelessComponent<IPreviewComponentProps> {
     return function BelgaCoveragePreview(props: IPreviewComponentProps) {
         const ids = parseIds(props.value);
 
-        console.info('ids', ids, props.value);
-
         return (
             <React.Fragment>
                 {ids.map((id) => (
                     <div key={id}>
-                        {props.item.type !== 'graphic' &&
-                            <BelgaCoverageInfo coverageId={id} superdesk={superdesk} />
-                        }
-                        <BelgaCoverageImages
-                            coverageId={id}
-                            rendition={'preview'}
-                            maxImages={10}
-                            superdesk={superdesk}
-                        />
+                        {props.item.type !== 'graphic' && (
```

Comment:
Is it really supposed to work for all types except graphic?

If so, it'd be nicer to use a ternary

```
{
    props.item.type === 'graphic'
        ? (
            <Component1 />
        )
        : (
            <Component2 />
        )
}
```